src/main/java/de/retest/recheck/persistence/bin/KryoPersistence.java

```diff
@@ -106,6 +126,15 @@ public T load( final URI identifier ) throws IOException {
 		}
 	}
 
+	private boolean isCompatible( final T persistable ) {
+		return isCompatible( persistable.getClass(), persistable.version() );
+	}
+
+	boolean isCompatible( final Class<? extends Persistable> clazz, final int version ) {
```

Comment:
We still have no guideline for this, but maybe a comment that says it is for testing? (I think Guava also has an annotation for this, at some point we might want to roll out our own `@TestingOnly`.)